_posts/2020-05-19-pathfinding-devlog.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Pathfinding for Enemy chase behavior, Difficulty with 2D navmashes
+description: Containing instructions for writing up posts for the development log.
+---
+
+# Development Log Dated 05-23-2020
+
+## Completed Milestones
+
+- Made an enemy navmesh agent that chases players around in the maze. (Victor)
+
+## The Problem
+
+Our original thinking for an enemy chasing a player through a maze was that it would be simple. Just use a navmesh and 
+navmesh agent, badabingbadaboom, task done in a couple of minutes. However, like most tasks in unity, it's not that simple.
+
+Turns out, navmesh agents aren't supported for 2D. After some minutes googling, it appears that there are a couple of 
+solutions available to me.
+
+1. Make the project 3D and pretend it's 2D, putting sprites on 3D objects. 
+Ok, cool, maybe next project, but we have done too much work in 2D at this point to convert everything.
+
+2. Import third party navmesh components for 2D.
+Open source is the best. Turns out other people have already solved this problem for me. There are a couple components out there,
+one we found in this thread: https://answers.unity.com/questions/1492606/unity-2d-navmesh-for-a-tilemap.html
+and the we decided to go with: https://github.com/h8man/NavMeshPlus
+
+NavMeshPlus by h8man is really well documented, and is ready to go by simply adding this dependency: 
+"com.h8man.2d.navmeshplus": "https://github.com/h8man/NavMeshPlus.git#master".
```

Comment:
Use triple backticks to format this as monospaced text. If it's a common language/syntax, you can reference it like so:

````
```unity
"com.h8man.2d.navmeshplus": "https://github.com/h8man/NavMeshPlus.git#master"
```
````

becomes:
```unity
"com.h8man.2d.navmeshplus": "https://github.com/h8man/NavMeshPlus.git#master"
```

````suggestion
```
"com.h8man.2d.navmeshplus": "https://github.com/h8man/NavMeshPlus.git#master"
```
````